server/controllers/client.ts

```diff
@@ -39,20 +40,47 @@ clientsRouter.use(
 )
 
 // Static HTML/CSS/JS client files
-
 const staticClientFiles = [
   'manifest.webmanifest',
   'ngsw-worker.js',
   'ngsw.json'
 ]
+
 for (const staticClientFile of staticClientFiles) {
   const path = join(root(), 'client', 'dist', staticClientFile)
 
-  clientsRouter.get('/' + staticClientFile, (req: express.Request, res: express.Response) => {
+  clientsRouter.get(`/${staticClientFile}`, (req: express.Request, res: express.Response) => {
     res.sendFile(path, { maxAge: STATIC_MAX_AGE.SERVER })
   })
 }
 
+// Static Logo/Favicon/Icons client images
+const staticClientImages = [
+  'logo.svg',
+  'favicon.png',
+  'icons/icon-36x36.png',
+  'icons/icon-48x48.png',
+  'icons/icon-72x72.png',
+  'icons/icon-96x96.png',
+  'icons/icon-144x144.png',
+  'icons/icon-192x192.png',
+  'icons/icon-512x512.png'
+]
+
+for (const staticClientImage of staticClientImages) {
+  let imagePhysicalPath = join(CONFIG.STORAGE.ASSETS_DIR, 'images', staticClientImage)
+  clientsRouter.use(`/client/assets/images/${staticClientImage}`, asyncMiddleware(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
please split the use and definition of the middleware.

Reply:
why not use `express.static` as well here?

Reply:
I thought it was better to only serve file we need, do you think we should be able to serve all the assets directory ?

Reply:
If the assets directory only has the files we need to serve, then yes :slightly_smiling_face: 

It has the advantage to be pretty rich when it comes to built-ins: https://expressjs.com/en/4x/api.html#express.static

Reply:
> The assets directory only has the files we need to serve, so yes

There are all assets of the theme, there is more than we want to serve, no ?

Or you means for the `icons` ? I think favicon and logo should serve alone.